src/awala.ts

```diff
@@ -1,9 +1,24 @@
 import { getModelForClass } from '@typegoose/typegoose';
 
-import type { SuccessfulResult } from './utilities/result.js';
+import type { Result, SuccessfulResult } from './utilities/result.js';
 import type { ServiceOptions } from './serviceTypes.js';
 import type { MemberBundleRequest } from './schemas/awala.schema.js';
 import { MemberBundleRequestModelSchema } from './models/MemberBundleRequest.model.js';
+import { MemberPublicKeyModelSchema } from './models/MemberPublicKey.model.js';
+import { addDays } from 'date-fns';
+import {
+  issueMemberCertificate,
+  retrieveVeraDnssecChain,
+  selfIssueOrganisationCertificate, serialiseMemberIdBundle,
+} from '@relaycorp/veraid';
+import { MemberModelSchema } from './models/Member.model.js';
+import { Kms } from './utilities/kms/Kms.js';
+import env from 'env-var';
+import { OrgModelSchema } from './models/Org.model.js';
+import { derDeserializePublicKey } from './utilities/derDeserialisePublicKey.js';
+
+const awalaEndpoint = env.get('AWALA_MW').asString();
```

Comment:
I think the word URL is important.

Reply:
I meant we should _add_ the word "URL", not necessarily drop "middleware". Both are key words.

`AWALA_MIDDLEWARE_URL`.